### ISSUE TITLE
Example with Tensorboard 2.0

### ISSUE THREAD

**Issue by prarshah** [rmeertens/Simplest-Tensorflow-Tensorboard-MNIST-Embedding-Visualisation#1]
Thank you so much for the simple and easy to understand explanation, however, I am facing some issues while running it with Tensorflow 2.0, will appreciate it if you could help with it.

**Comment by rmeertens** [comment]
what are the issues in running it with tensorflow 2.0? What did you try and what were the expected results? Are any functions deprecated or changed?

**Comment by prarshah** [comment]
Thank you for the quick reply.
Yes, many functionalities like `tensorflow.contrib.tensorboard.plugins import projector`  have changed, even though I followed StackOverflow to solve such issues in eager mode, I am still working on it to write a generic function so that given an input dataset it will generate a projection. 
Eventually, I may be able to complete it but I am still not sure if it is the correct way to do it as no docs are available yet and there are hardly any well-explained examples for 2.0 like yours.

**Comment by rmeertens** [comment]
Can you maybe look at this issue? It seems to contain a lot of valuable information: https://github.com/tensorflow/tensorboard/issues/2471. 

Also: please send me a notebook in case you managed to get it working :)

**Comment by prarshah** [comment]
Sure, thanks again :)

**Comment by rmeertens** [comment]
Hey Prashah, quick question. It looks that there is now a hosted tensorboard from tensorflow itself. You can find it here: http://projector.tensorflow.org/

This code seems to generate correct vectors you can use to visualise it: 
`%matplotlib inline
import matplotlib.pyplot as plt
import tensorflow as tf
import numpy as np
import os
import json
import numpy as np

TO_EMBED_COUNT = 10000

(x_train, y_train), (x_test, y_test) = tf.keras.datasets.mnist.load_data()
batch_xs, batch_ys = x_train[:TO_EMBED_COUNT, :], y_train[:TO_EMBED_COUNT]

with open("vectors.tsv", 'w') as f:
    for image in batch_xs:
        image = image.flatten()
        f.write("\t".join([str(a) for a in image]))
        f.write("\n")
with open('labels.tsv', 'w') as f:
    for label in batch_ys: 
        f.write(str(label) + "\n")`


Does that help you?

**Comment by prarshah** [comment]
Yes, it is exemplary and it actually solved most of my problem! Thank you.
